### PR TITLE
Updated maven to 3.6.1 and installed OpenJDK 6-11 built by Azul Syste…

### PR DESCRIPTION
**Pull request by aberman**
Updated maven to 3.6.1 and installed OpenJDK 6-11 built by Azul Systems, Inc.

### CODE REVIEW

**Comment by ryandens** [comment]
should we rename this project to be maven-java-6-through-11?

**Comment by tylernhoward** [comment]
@ryandens Yes, we should come up with a more inventive name. `maven-cross-openjdk-docker`?

File: README.md [hunk 1]
```diff
@@ -1,2 +1,4 @@
 # maven-java7-java8
-Dockerfile which builds an image containing maven, openjdk7, and openjdk8
+Dockerfile which builds an image containing Maven 3.6.1, OpenJDK 6, OpenJDK 7, OpenJDK 8, and OpenJDK 11.  All OpenJDKs are built and supported by Azul Systems, Inc. [Click here for more Information](https://www.azul.com/downloads/zulu/).  
+
+Azul Systems, Inc. runs the OpenJDK code through the full JCK/TCK to ensure that it complies with the Java SE spec, as defined by the JCP/JSR. In addition, they also run an analysis using tools they've developed to ensure that every single file that is compiled (both static and dynamically generated during the build) has the correct GPLv2 with CPE license header. This way they can also guarantee that there are no licensing issues.
```
Comment (tylernhoward) [inline README.md:4]: I have never heard of this before. This is apples to oranges to the openJDK?

File: Dockerfile [hunk 2]
```diff
@@ -1,19 +1,17 @@
...
-    echo 'deb http://deb.debian.org/debian jessie-backports main' \
-      > /etc/apt/sources.list.d/jessie-backports.list && \
-    apt-get update -y && \
-    apt-get install -t \
-      jessie-backports \
-      openjdk-8-jdk \
-      ca-certificates-java -y && \
-    # Remove old symlink to set openJDK-8 as default
-    update-alternatives --remove java /usr/lib/jvm/java-7-openjdk-amd64/jre/bin/java
+    apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 0xB1998361219BD9C9 && \
```
Comment (tylernhoward) [inline Dockerfile:3]: `hkp://keyserver.ubuntu.com:80 --recv-keys 0xB1998361219BD9C9` - is this okay to stick in a public repository? Unfamiliar with using keyservers.

File: Dockerfile [hunk 3]
```diff
@@ -1,19 +1,17 @@
...
+    echo "deb http://repos.azulsystems.com/debian stable main" > /etc/apt/sources.list.d/zulu.list && \
+    apt-get autoremove -y openjdk-11-jre-headless openjdk-11-jdk && \
```
Comment (tylernhoward) [inline Dockerfile:5]: why do we have to remove this? is it bundled somewhere in the imported image?

**Comment by tylernhoward** [comment]
- [x] rename repository

**Comment by ryandens** [comment]
> @ryandens Yes, we should come up with a more inventive name. `maven-cross-openjdk-docker`?

I like this name far better 👍

**Review by tylernhoward — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -1,2 +1,4 @@
 # maven-java7-java8
```
Comment (tylernhoward) [inline README.md]: change this to `maven-cross-openjdk-docker`

**Review by tylernhoward — CHANGES_REQUESTED** [review]

**Comment by tylernhoward** [comment]
> > @ryandens Yes, we should come up with a more inventive name. `maven-cross-openjdk-docker`?
> 
> I like this name far better 👍

Done

File: README.md (hunk 1 above)
Comment (aberman) [reply README.md:4]: It is a certified and fully supported build of openjdk.  It's like Amazon Corretto but not Amazon :-)

File: Dockerfile (hunk 2 above)
Comment (aberman) [reply Dockerfile:3]: Yes, this is a public key that downloads to verify the package.

File: Dockerfile (hunk 3 above)
Comment (aberman) [reply Dockerfile:5]: Yes, it's included in the image.  I nuked it for a couple reasons: consistent use of the Azul JDKs and to reduce the size of the image.

**Review by tylernhoward — APPROVED** [review]

**Pull request merged**